features/fixtures/news_and_communications.json

```diff
@@ -89,11 +89,11 @@
         "preposition": "about"
       },
       {
-        "key": "related_to_brexit",
+        "key": "related_to_transition_period",
```

Comment:
Are we catering for redirects?  We know that people have RSS feeds set up for this, and changing the key will stop these from working unless we do something about it.

Reply:
ah... humm.... that's a really good shout and a pain...
Ideally we don't want any user facing references to brexit anymore as all this content should now be about transition.

... can we redirect on a param?

Reply:
Humm ok, rolling this back for now in an effort to separate out the content change from the key change.